src/lib/operators/validate.hpp

```diff
@@ -26,11 +26,14 @@ class Validate : public AbstractReadOnlyOperator {
   static bool is_row_visible(TransactionID our_tid, CommitID snapshot_commit_id, const TransactionID row_tid,
                              const CommitID begin_cid, const CommitID end_cid);
 
+  static bool is_entire_chunk_visible(const std::shared_ptr<const Chunk>& chunk, const CommitID snapshot_commit_id);
```

Comment:
Doc? Also explain that this is only valid after checking `can_use_chunk_shortcut`.

Reply:
I could also imagine to make `is_entire_chunk_visible` a private non-static method and `can_use_chunk_shortcut` a member and have an assert on `can_use_chunk_shortcut`.